.eslintrc.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
         'plugin:@typescript-eslint/recommended',
         'plugin:prettier/recommended',
         'prettier',
-        'prettier/@typescript-eslint',
```

Comment:
this is to fix eslint?

Reply:
Yes, apparently you are not allowed to do that anymore. Everything is covered by the 'prettier' element.